rollup.config.js

```diff
@@ -102,6 +118,9 @@ const plugins = [
   postcss({
     exclude: ['**/*.mod.css'],
     include: ['**/*.css'],
+    // literally the only reason we have this, is because of the `select.css` file
+    // or else we would not need to include any plugins as this would
+    // just be vendor (plain) css.
```

Comment:
Is there any way to get rid of it as well?

Reply:
Yes, what I would like to do is in the next step, export a javascript file with all of our css variables, and then remove `select.css` and replace it by importing this file into [select-input](https://github.com/commercetools/ui-kit/blob/0a0722a2d28d13901ed8d2664df39658c29918c4/src/components/inputs/select-input/select-input.js) and passing the styles with the design tokens into react-select there.